client/asset/eth/eth.go

```diff
@@ -88,7 +88,7 @@ const (
 	// confCheckTimeout is the amount of time allowed to check for
 	// confirmations. Testing on testnet has shown spikes up to 2.5
 	// seconds. This value may need to be adjusted in the future.
-	confCheckTimeout = 4 * time.Second
```

Comment:
We are a little worried about trade ticks taking a long time, and this may increase those. Do the errors end in failed trades or they are just loud?

Actually the comment is probably from back when we thought we could use light nodes, so providers would be longer. 30 seconds feels excessive though. Does just a little more work? like... 6 or 8?

Reply:
> could maybe try 15 or 20 though (testing, will report back in several days or 1 week)

I've been running this change with 15s (instead of 30s) for about 5 days now, it seems to work just as flawlessly for me so far - but maybe that's just me (my network, and location)

15s might not be enough for Tor-based connections though for example, so I personally think 30s is more reasonable timeout value for www-request

as I mentioned above, trying to tweak this timeout lower for the sake of **trade ticks** seems like a wrong way to go about it (the interaction with trade ticks needs to be handled differently, not the timeout)

cc @buck54321 @martonp if you have any opinion on this ^

Reply:
I've long considered some kind of system where the default is something long like 1 - 5 minutes, but if there are pending transactions, or if the user takes some action that indicates that they may be waiting for a transaction, such as calling `FundOrder` or opening the "Receive" dialog in the UI, we signal the wallet to check more frequently, like every 10 or 20 seconds, for a while.

There are also probably some improvements to be made around the handling of "ticks" in **core**. A good audit of the synchronicity patterns in those routines would be helpful.

Reply:
I think 30 seconds is fine for now. I have seen this notification as well and it's a bit annoying.